slowfast/models/video_model_builder.py

```diff
@@ -468,7 +472,7 @@ def _construct_network(self, cfg):
             norm_module=self.norm_module,
         )
 
-        self.s2 = resnet_helper.ResStage(
+        s2 = resnet_helper.ResStage(
```

Comment:
Thanks a lot for enabling checkpoint here! We might want to keep the original name of `self.xx` for non-checkpointing cases for backward compatibility, more specifically for checkpoints here:
https://github.com/facebookresearch/SlowFast/blob/master/MODEL_ZOO.md

Reply:
Good point! I haven't tried keeping s1 and s2 attached to self. I remember in the past it can cause problems in some cases when there are multiple "paths" to reach the same module from the parent module. But I am not sure it will be an issue here. Let me update this diff to keep s1/s2 in the original place.

Reply:
done.